tools/fiptool/fiptool.c

```diff
@@ -52,34 +52,6 @@
 #define OPT_PLAT_TOC_FLAGS 1
 #define OPT_ALIGN 2
 
-static image_desc_t *lookup_image_desc_from_uuid(const uuid_t *uuid);
```

Comment:
Right.  I had noticed that.  drivers/io/* are not nice.  They need much reworking.  I like to avoid declarations for static functions.   There is no point to describe the same thing twice.